config/db_tools.standalone.sample.yaml

```diff
@@ -1,130 +1,127 @@
-workdir: /var/www/my_project/
-
-# Database connections.
-# One line per connection, a single database URL, all options as query
-# parameters. Connections will be made using makincorpus/query-builder
-# which will raise exceptions when invalid options are found.
-# There is less configuration amplitude than using doctrine/dbal in
-# Symfony, yet it should be enough in most case.
-# In case any options or specific behaviour is missing, please file
-# an issue at https://github.com/makinacorpus/php-query-builder/issues
-connections:
-    connection_one: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
-    connection_two: "mysql://username:password@hostname:port?version=8.1&other_option=..."
-
-# If you have a single connection, you can use this syntax. In this case
-# the connection name will be "default".
-# connections: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
+# In most cases, you will not need it.
+#workdir: .
 
-# You can explicitely set which will be default connection in use when
-# none providen in the command line options. If you omit this configuration
-# value, then the first one in the list will be used.
-#default_connection: connection_one
+# Where to put generated backups.
+# Root directory of the backup storage manager. Default filename
+# strategy will always use this folder as root path.
+#storage_directory: ./var/db_tools
 
-# Using the DbToolsBundle standalone, you must provide at least
-# a root directory for backups.
-storage:
-    # Path can be relative or absolute, Relative paths are relative to the
-    # workdir option if specified, or from this configuration file directory
-    # otherwise.
-    # If none provided, the default will be the following one.
-    root_dir: ./var/db_tools
-    # Filename strategies. You may specify one strategy for each doctrine
-    # connection. Keys are doctrine connection names. Values are strategy
-    # names, "default" (or null) or omitting the connection will use the
-    # default implementation.
-    # If you created and registered a custom one into the container as a
-    # service, you may simply set the service identifier. If no service
-    # exists, and your implementation does not require parameters, simply
-    # set the class name.
-    #filename_strategy:
-        # Backup filename strategy.
-        # "default" is an alias of "datetime"
-        #default: default
-        # "datetime" implementation is ROOT_DIR/YYYY/MM/<connection-name>-<datestamp>.<ext>"
-        #other_connection_strategy: datetime
-        # Example of using a service name:
-        #yet_another_connection: app.db_tools.filename.custom_strategy
-        # Or a classe name:
-        #another_one: App\DbTools\Storage\MyCustomStrategy
-
-# When old backups are considered obsolete
+# Default filename strategy. You may specify one strategy for each doctrine
+# connection.
+# If you created and registered a custom one into the container as a
```

Comment:
```suggestion
# If you created and registered a custom filename strategy into the container as a
```

Not sure about all this paragraph, it reads like Symfonishy. How it works for standalone version ?